index.html

```diff
@@ -0,0 +1,408 @@
+                    </li>
+                    <li>English</li>
+                    <li>My page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="col-2"></div>
+        </div>
+
+        <!--Lower navigation-->
+        <div class="row lower_navigation">
+            <div class="col-1"></div>
+            <div class="col-2 about_logo">
+                <a href="index.html">
+                    <img src="images/women.png" alt="Logo Image">
+                </a>
+            </div>
+            <div class="col-1"></div>
+            <div class="col-8">
+                <ul>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="##">Program</a>
+                    </li>
+                    <li>
+                        <a href="##">Join</a>
+                    </li>
+                    <li>
+                        <a href="##">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="##">News</a>
+                    </li>
+                    <li>
+                        <a href="##">Follow us</a>
+                    </li>
+                </ul>
+            </div>
+
+        </div>
+
+
+
+
+
+        <!--main section-->
+        <main class="container-fluid main-section">
+            <div class="container hambuger">
+                <div class="row">
+                    <div class="col-1 hambuger-menu">
+                        <img src="./images/mobile_menu.png" alt="Menu bar image">
+
+                    </div>
+                </div>
+
+                <!--Hamburger drop down-->
+                <div class="container-fluid hide-menu">
+                    <div class="row">
+                        <div class="col-12 drop-down-menu">
+
+                            <ul>
+                                <li>
+                                    <a href="about.html">About</a>
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
+                            </ul>
+                            <span class="close_button">&times;</span>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [Optional] Please use one navigation list or mobile and desktop version 👌👌
- [Optional] Kindly make the navigation inside the `header` and `nav` tag for readability and search engines👌